src/model_loader_assimp.cpp

```diff
@@ -0,0 +1,348 @@
+#include "model_loader_assimp.hpp"
+
+#include "util/log.hpp"
+
+namespace wr
+{
+
+	AssimpModelLoader::AssimpModelLoader()
+	{
+		m_supported_model_formats.push_back("obj");
```

Comment:
Initializer list could be better. More readable and probably faster (Viktor said)